npm/private/npm_import.bzl

```diff
@@ -33,7 +33,7 @@ def npm_imported_package_store(
     ref_deps = {ref_deps}
 
     has_lifecycle_build_target = {has_lifecycle_build_target}
-    store_target_name = "{virtual_store_root}/{{}}/{package}/{version}".format(link_root_name)
```

Comment:
Was there previously any reason that this was *not* `virtual_store_name`?

Reply:
Legacy reasons I believe when the virtual store name needed to be friendly. It doesn't any more so the target name can match the directory name which is much better.